README.md

```diff
@@ -74,6 +74,41 @@ webpi_product 'WindowsAzurePowerShellGet' do
 end
 ```
 
+
+### webpi_application
+
+#### Actions
+
+- :install: install an application using WebpiCmdLine
+
+#### Attribute Parameters
+
+- app_id: name attribute. Specifies the ID of an application to install.
+- accept_eula: specifies that WebpiCmdline should auto-accept EULAs. Default is false.
+- returns: specifies the return value(s) expected for a successful installation. Can be a single integer or array of integers. Default is [0, 42]
```

Comment:
can you document the other properties. Not all of them are in this readme

Reply:
I have added decent documentation for the properties